### PR TITLE
bridge: Add internal D-Bus API to read cockpit.conf

### PR DESCRIPTION
**Pull request by martinpitt**
Add a new cockpit.Config.Get{String,UInt}() methods on the internal
bridge D-Bus server that provides an interface to
`cockpit_conf_{string,uint}()` for JavaScript.

This will be used for implementing the customization of the automatic
logout on idle.

### CODE REVIEW

**Comment by martinpitt** [comment]
@marusak: This is what we talked about today. Does this make sense to you?

**Comment by martinpitt** [comment]
@mvollmer: @marusak confirmed that he can use this for the screen timeout. Do you have some time to review this?

File: src/bridge/cockpitdbusconfig.c [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+{
+  if (g_str_equal (method_name, "GetString"))
+    {
+      const gchar *section, *key;
+      const char *result;
+
+      g_variant_get (parameters, "(&s&s)", &section, &key);
+
+      /* Ensure we don't work from an old cache, force re-reading the file */
+      cockpit_conf_cleanup ();
```
Comment (mvollmer) [inline src/bridge/cockpitdbusconfig.c]: I wouldn't expect the config to be reloaded for every GetString or GetUInt call.  Right now, cockpit-ws never reloads its configuration and I don't think we should change that.  If necessary for the tests, let's introduce a new Reload method.

Comment (martinpitt) [reply src/bridge/cockpitdbusconfig.c]: Makes sense, updated to do this. I also fixed the `QUnit.only` leftover, oops!

**Pull request merged**